Add `Corgi Cup: June 2016` tournament article (#5723)
ppy/osu-wiki: wiki/Tournaments/CC/1/en.md

**Corgi Cup: June 2016** (***CC***) was a 1v1 single-elimination osu! tournament hosted by ![][flag_FI] [M8gazine](https://osu.ppy.sh/users/5039710), ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983), and ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973). It was the first installment of the Corgi Cup.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2016-06-02/2016-06-16 |
| Qualifiers | 2016-06-17 |
| Round of 16 | 2016-06-18 |
| Quarterfinals | 2016-06-19 |
| Semifinals | 2016-06-19 |
| Grand Finals | 2016-06-19 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter tag, Corgi Cup 1st place Discord role |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 3 months of osu!supporter tag |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag |

## Organisation

Corgi Cup: June 2016 was run by various community members by distributing the multitude of tasks into various fields of responsibility.

| Position | Member(s) |
| :-- | :-- |
| Host | ![][flag_FI] [M8gazine](https://osu.ppy.sh/users/5039710) |

> | Co-Host | ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) |

Reviewer: ```suggestion
| Co-host | ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) |
```

  - [Sako Tomohisa - Kirakira (Dino99) [Extra]](https://osu.ppy.sh/beatmapsets/449527#osu/964586)
  - [High Contrast - The Agony & the Ecstasy (Feat. Selah Corbin) (samplefish) [Ecstasy]](https://osu.ppy.sh/beatmapsets/192342#osu/457448)
- Hidden
  - [Dio ft. Sef - Tijdmachine (GladiOol) [Lesjuh! '11]](https://osu.ppy.sh/beatmapsets/6997#osu/129875)
  - [An - Xhroria (Cherry Blossom) [Another]](https://osu.ppy.sh/beatmapsets/82258#osu/227772)
- HardRock
  - [Shawn Wasabi - Marble Soda (Len) [Crier's Extra]](https://osu.ppy.sh/beatmapsets/387784#osu/846259)
  - [S-C-U - concon (Sync) [Extreme]](https://osu.ppy.sh/beatmapsets/42051#osu/133053)
- DoubleTime
  - [HTT - Pure Pure Heart (miccoliasms) [miccoli]](https://osu.ppy.sh/beatmapsets/16523#osu/67469)
  - [Sakura Chiyo (CV:Ozawa Ari) - URAOMOTE FORTUNE (P A N) [Insane]](https://osu.ppy.sh/beatmapsets/210706#osu/495590)
- FreeMod
  - [Telekinesis - The Monk (FCL) [Technique of Shadow]](https://osu.ppy.sh/beatmapsets/381529#osu/834589)
  - [IRON ATTACK! - Future is Undefined (Kite) [Insane]](https://osu.ppy.sh/beatmapsets/32447#osu/106156)
- Tiebreaker

>   - **[SawanoHiroyuki\[nZk\]:Gemie - x.U. (ETIA. Hardcore Bootleg Remix) (captin1) [Shinoa]](https://osu.ppy.sh/beatmapsets/331872#osu/735021)**  

Reviewer: ```suggestion
  - **[SawanoHiroyuki[nZk]:Gemie - x.U. (ETIA. Hardcore Bootleg Remix) (captin1) [Shinoa]](https://osu.ppy.sh/beatmapsets/331872#osu/735021)**  
```

### Quarterfinals

- NoMod
  - [Nekomata Master - Far east nightbird (kors k Remix) (jonathanlfj) [RLC's Extra]](https://osu.ppy.sh/beatmapsets/144171#osu/358273)
  - [Traktion - Gid V (Irreversible) [galV's Insane]](https://osu.ppy.sh/beatmapsets/163689#osu/399252)

>   - [Dark PHOENiX - Stirring an Autumn Moon (_lolipop) [Crazy Moon]](https://osu.ppy.sh/beatmapsets/16650#osu/59693)

Reviewer: ```suggestion
  - [Dark PHOENiX - Stirring an Autumn Moon (\_lolipop) [Crazy Moon]](https://osu.ppy.sh/beatmapsets/16650#osu/59693)
```

| **rafalbrew** ![][flag_GB] | **4** | 1 | ![][flag_GB] Pelzio | [#1](https://osu.ppy.sh/community/matches/25744050) |

#### Third place match

Sunday, 19 June 2019:

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **zelda101** ![][flag_US] | **4** | 0 | ![][flag_GB] dong | [#1](https://osu.ppy.sh/community/matches/25744734) |

### Semifinals

Sunday, 19 June 2019:

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| zelda101 ![][flag_US] | 2 | **4** | ![][flag_GB] **rafalbrew** | [#1](https://osu.ppy.sh/community/matches/25742222) |
| **Pelzio** ![][flag_GB] | **4** | 3 | ![][flag_GB] dong | [#1](https://osu.ppy.sh/community/matches/25743314) |

### Quarterfinals

Sunday, 19 June 2019:

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Seraxis ![][flag_GB] | 3 | **4** | ![][flag_US] **zelda101** | [#1](https://osu.ppy.sh/community/matches/25737972) |
| **Pelzio** ![][flag_GB] | **4** | 1 | ![][flag_US] MicMck101 | [#1](https://osu.ppy.sh/community/matches/25739013) |
| **dong** ![][flag_GB] | **4** | 3 | ![][flag_GB] djsettle | [#1](https://osu.ppy.sh/community/matches/25741329) |

> | Hydria ![][flag_GB] | 2 | **4** | ![][flag_GB] rafalbrew | [#1](https://osu.ppy.sh/community/matches/25740307) |

Reviewer: ```suggestion
| Hydria ![][flag_GB] | 2 | **4** | ![][flag_GB] **rafalbrew** | [#1](https://osu.ppy.sh/community/matches/25740307) |
```

### Round of 16

Saturday, 18 June 2019:

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Atrocious ![][flag_SE] | 3 | **4** | ![][flag_GB] **dong** | [#1](https://osu.ppy.sh/community/matches/25710268) |

> | Sopison ![][flag_GB] | 0 | **4** | ![][flag_GB] djsettle | [#1](https://osu.ppy.sh/community/matches/25711194) |

Reviewer: ```suggestion
| Sopison ![][flag_GB] | 0 | **4** | ![][flag_GB] **djsettle** | [#1](https://osu.ppy.sh/community/matches/25711194) |
```

| **Pelzio** ![][flag_GB] | **4** | 1 | ![][flag_GB] MagicMantwon | [#1](https://osu.ppy.sh/community/matches/25711910) |
| Lazulite ![][flag_US] | 3 | **4** | ![][flag_GB] **Hydria** | [#1](https://osu.ppy.sh/community/matches/25712820) |
| **rafalbrew** ![][flag_GB] | **4** | 0 | ![][flag_GB] SteffoST | [#1](https://osu.ppy.sh/community/matches/25713279) |
| **Seraxis** ![][flag_GB] | **4** | 0 | ![][flag_CZ] NyarkoO | [#1](https://osu.ppy.sh/community/matches/25715231) |
| **zelda101** ![][flag_GB] | **4** | 1 | ![][flag_DE] Blegie | [#1](https://osu.ppy.sh/community/matches/25715822) |
| YouKittenMe ![][flag_GB] | 2 | **4** | ![][flag_US] **MicMck101** | [#1](https://osu.ppy.sh/community/matches/25716816) |

## Ruleset

> - This tournament is a 1v1, Head to head, Single Elimination tournament.

Reviewer: ```suggestion
- This is a 1v1, head-to-head, single-elimination tournament.
```

- Each round will be BO7 (first to 4).
- Referees create the match and will invite players 10 mins prior to match beginning.
- If a player fails to turn up before or on the time their match is due to start, a 10 minute timer will be started. After 10 minutes, if the player has still not shown up, the other player wins.
- If both players fail to show up, the match can be rescheduled during the times given.
- Warm-ups are optional should a player request one. (Note: the warm up should not be from the mappool the players are due to play and must have a drain time of 4 mins or less.)
- After warm ups (should there be any), both players will `!roll` to decide who bans/picks first. Loser of the roll bans first and winner of the roll picks first.

> - Each player is allowed one ban per match (Not Tiebreaker).

Reviewer: ```suggestion
- Each player is allowed one ban per match. The tiebreaker cannot be banned.
```

- If a player disconnects during the match in which the outcome would have been different. A rematch can be offered under the Referee’s discretion.

> - Hard Rock and Double Time mappools can be played with Hidden should the player want to.

Reviewer: ```suggestion
- Hard Rock and Double Time mappools can be played with Hidden if the player wants to.
```

- Score V2 will be used.

> - Free mod can be any mod the player decides (including EZ + FL).

Reviewer: ```suggestion
- Free Mod can be any mod the player decides (including EZ + FL).
```
